File: 3636-check-balanced-string/3636-check-balanced-string.cpp
class Solution {
public:
    bool isBalanced(string num) {
        long long odd=0;
        long long even=0;
        for (int i=0;i<num.length();){
            odd+=num[i]-'0';
            i+=2;
        }
        for (int i=1;i<num.length();){
            even+=num[i]-'0';
            i+=2;
        }

        return even == odd;
            
        
    }
};

File: 3636-check-balanced-string/3636-check-balanced-string.py
class Solution:
    def isBalanced(self, num: str) -> bool:
        odd=0
        even=0
        for i in range(0,len(num),2):
            odd+=int(num[i])
        for i in range(1,len(num),2):
            even+=int(num[i])
        return even == odd
            